File: customers-staff-master/src/Front/src/api/new/dto/WorkOrder/WorkOrderItemDto.ts
import { DateTime } from "../DateTime";
import { Guid } from "../Guid";
import { WorkOrderNumberDto } from "./WorkOrderNumberDto";
import { WorkOrderStatusDto } from "./WorkOrderStatusDto";

export interface WorkOrderItemDto {
  id: Guid;
  number: WorkOrderNumberDto;
  status: WorkOrderStatusDto;
  receptionDate: DateTime;
  totalSum: number;
  firstProductId?: Guid;
  clientId: Guid;
}

File: customers-staff-master/src/CustomersAndStaff/FrontApi.Dto/WorkOrders/WorkOrderItemDto.cs
using System;

namespace Market.CustomersAndStaff.FrontApi.Dto.WorkOrders
{
    public class WorkOrderItemDto
    {
        public Guid Id { get; set; }
        public WorkOrderNumberDto Number { get; set; }
        public WorkOrderStatusDto Status { get; set; }
        public DateTime ReceptionDate { get; set; }
        public decimal TotalSum { get; set; }
        public Guid? FirstProductId { get; set; }
        public Guid ClientId { get; set; }
    }
}